apps/webapp/app/routes/engine.v1.worker-actions.deployments.$deploymentFriendlyId.dequeue.ts

```diff
@@ -14,55 +13,7 @@ export const loader = createLoaderWorkerApiRoute(
       maxRunCount: z.coerce.number().optional(),
     }),
   },
-  async ({
-    authenticatedWorker,
-    params,
-    searchParams,
-  }): Promise<TypedResponse<WorkerApiDequeueResponseBody>> => {
-    const deployment = await $replica.workerDeployment.findUnique({
-      where: {
-        friendlyId: params.deploymentFriendlyId,
-      },
-      include: {
-        worker: true,
-      },
-    });
-
-    if (!deployment) {
-      throw new Error("Deployment not found");
-    }
-
-    if (!deployment.worker) {
-      throw new Error("Worker not found");
-    }
-
-    const dequeuedMessages = (await isCurrentDeployment(deployment.id, deployment.environmentId))
-      ? await authenticatedWorker.dequeueFromEnvironment(
-          deployment.worker.id,
-          deployment.environmentId
-        )
-      : await authenticatedWorker.dequeueFromVersion(
-          deployment.worker.id,
-          searchParams.maxRunCount
-        );
-
-    return json(dequeuedMessages);
+  async (): Promise<TypedResponse<WorkerApiDequeueResponseBody>> => {
+    return json([]);
```

Comment:
@ericallam does this mean when we put the platform live, we'll need to very quickly deploy the new supervisor as well otherwise no v4 runs will get dequeued?